config/rbac/role.yaml

```diff
@@ -5,18 +5,6 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
-- apiGroups:
-  - cinder.openstack.org
-  resources:
-  - cinders
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
```

Comment:
Did you mean to change the cinder permissions? perhaps a rebase is needed here?

Reply:
+1 seems rebase missing + ovs permissions also missing as per
//+kubebuilder:rbac:groups=ovs.openstack.org,resources=ovs,verbs=get;list;watch;create;update;patch;delete
//+kubebuilder:rbac:groups=ovs.openstack.org,resources=ovs/status,verbs=get;update;patch

Reply:
yes, sorry for that mistake. Fixed now